app/controllers/idv/by_mail/request_letter_controller.rb

```diff
@@ -60,7 +60,7 @@ def confirm_letter_sends_allowed
 
       def step_indicator_steps
         if in_person_proofing?
-          Idv::Flows::InPersonFlow::STEP_INDICATOR_STEPS_GPO
+          Idv::StepIndicatorConcern::STEP_INDICATOR_STEPS_IPP
         else
```

Comment:
I have not tested but I am wondering if this if statement should also be deleted?  (Below is resend the letter- I think this is the original send letter)